src/modules/projectSearchContext.js

```diff
@@ -125,14 +125,14 @@ class ProjectSearchContext {
         }
         const searchRequest = typeof request === 'string' ? {query: {human: request}} : request;
 
-        return this.sessionManager.gatekeeper(`/${this.collection}/${this.collectionId}/traces/search`, {
+        return this.sessionManager.gatekeeper(`/projects/${this.collectionId}/traces/search`, {
```

Comment:
I guess the logic differs too much between `singlefiles` and `projects` to keep them in a module that caters to both?